sympy/pm/pm.py

```diff
@@ -0,0 +1,64 @@
+from sympy.utilities.iterables import dict_merge
+from sympy.core.compatibility import _nodes
+
+
+def rubi_match(subject, pattern, var=None):
```

Comment:
Why `rubi_match`? If we want to create a matching object, it should not be restricted to the RUBI only.